project.clj

```diff
@@ -30,6 +34,8 @@
   :classifiers [["test" :testutils]]
 
   :profiles {:dev {:dependencies [[spyscope "0.1.4"]
```

Comment:
Yes please.  Good to hear you use it, we'll know if we actually *can* get rid of it then!  (We could the last time I tried a few months ago)